javalin/src/main/java/io/javalin/Javalin.java

```diff
@@ -15,11 +15,7 @@
 import io.javalin.event.HandlerMetaInfo;
 import io.javalin.event.JavalinEvent;
 import io.javalin.event.WsHandlerMetaInfo;
-import io.javalin.http.Context;
-import io.javalin.http.ExceptionHandler;
-import io.javalin.http.Handler;
-import io.javalin.http.HandlerType;
-import io.javalin.http.JavalinServlet;
+import io.javalin.http.*;
```

Comment:
You should configure IntelliJ to not group into star imports.

Reply:
I have already done it, this changes might be from an early commit, will revert any star grouping.

Reply:
Fixef